src/acuvity/apexdiscovery.py

```diff
@@ -1,10 +1,14 @@
-import jwt
+from typing import Any, Callable, Optional, Tuple, Union
 from urllib.parse import urlparse
-from typing import Callable, Optional, Union, Tuple, Any
-from .httpclient import HttpClient
+
+import jwt
+from pydantic import BaseModel
+
 from acuvity import models
 from acuvity.utils import get_security_from_env
-from pydantic import BaseModel
+
+from .httpclient import HttpClient
+
```

Comment:
doesn't look like any of these imports are required. This will fail the SDK generation. Please test with the speakeasy CLI